Detectors/TOF/simulation/src/Detector.cxx

```diff
@@ -1861,27 +1867,29 @@ void Detector::addAlignableVolumes() const
       volPath += vpL4;
       volPath += istr;
 
-      volPath = "";
-
       symName = snSM;
       symName += Form("%02d", isect);
       symName += snSTRIP;
       symName += Form("%02d", istr);
```

Comment:
I would suggest to have/use static member methods to get the symbolic names of alignable volumes, e.g.
``
static TString getSMSymbolicName(int sm) {return TString::Format("TOF/sm%02d",sm));}
static TString getStripSymbolicName(int sm,int st) {return TString::Format("TOF/sm%02d/strip%02d",sm,st));}
``
since eventually external code will need to know how to access the volumes by their symbolic name.